xtask/README.md

```diff
@@ -9,7 +9,8 @@ the appropriate fields.
 
 ## Commands
 
-- `release [crate]`: Publish `crate`, create a signed tag based on its name and version and create
-  a release on GitHub. **Requires all `github` fields in `config.toml`.**
+- `release [crate] [version]`: Publish `crate` at given `version`, create a signed tag based on its
```

Comment:
```suggestion
- `release [crate] [version]`: Publish `crate` at given `version`, if applicable¹ create a signed tag based on its
```

and

```markdown
¹ if `crate` is a user-facing crate and `version` is not a pre-release
```

or something like that

Reply:
Actually I was wondering seeing the past releases. It seems like the only difference between release and pre-release is that the changelog is not updated with the latter. Right now it stops at publishing.

Reply:
Hm yeah I guess we do have some pre-release tags too. I think I don't want to generate notifications for those for people watching releases (since excluding pre-releases in that is not a config option GitHub has), so the GitHub API step should definitely be left out.